app/src/main/java/com/github/onlynotesswent/ui/user/EditProfile.kt

```diff
@@ -228,27 +217,47 @@ fun ProfilePicture(
         contentScale = ContentScale.Crop)
 
     // Edit Icon Overlay
-    Icon(
-        imageVector = Icons.Default.Edit,
-        contentDescription = "Edit Profile Picture",
+    IconButton(
+        onClick = { // Edit the image and save the URI to the profilePicture state
+          profilePictureTaker.setOnImageSelected { uri ->
+            if (uri != null) {
+              profilePictureUri.value = uri.toString()
+              hasProfilePictureBeenChanged.value = true
+            }
+          }
+          profilePictureTaker.pickImage()
+        },
         modifier =
             Modifier.testTag("editProfilePicture")
-                .size(40.dp) // Size of the edit icon
-                .align(Alignment.BottomEnd) // Position on the bottom-left corner
-                .offset(x = (-8).dp, y = (-8).dp)
-                .clip(CircleShape)
-                .background(Color.White) // Background color
-                .clickable {
-                  // Edit the image and save the URI to the profilePicture state
-                  profilePictureTaker.setOnImageSelected { uri ->
-                    if (uri != null) {
-                      profilePictureUri.value = uri.toString()
-                      hasProfilePictureBeenChanged.value = true
-                    }
-                  }
-                  profilePictureTaker.pickImage()
-                },
-        tint = Color.Gray // Icon color
-        )
+                .align(Alignment.CenterEnd) // Position on the bottom-right corner
+                .offset(x = 42.dp, y = 15.dp)
+                .clip(CircleShape)) {
+          Icon(
+              imageVector = Icons.Default.Edit,
+              contentDescription = "Edit Profile Picture",
+              modifier = Modifier.testTag("editProfilePicture").size(40.dp) // Size of the edit icon
+              ,
+              tint = Color.Blue // Icon color
+              )
+        }
+
+    // Delete Image Icon
+    IconButton(
```

Comment:
Not sure if it's necessary to be able to delet a profile picture

Reply:
yes it is necessary